examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,266 @@
+n_time_bins = 35
+times = t0 + np.arange(n_time_bins) * duration
+time_intervals = [Time([tstart, tstop]) for tstart, tstop in zip(times[:-1], times[1:])]
+print(time_intervals[0].mjd)
+
+short_observations = observations.select_time(time_intervals)
+
+######################################################################
+# check that observations have been filtered
+
+print(f"Number of observations after time filtering: {len(short_observations)}\n")
+print(short_observations[1].gti)
+
+######################################################################
+# Target definition
+energy_axis = MapAxis.from_energy_bounds("0.4 TeV", "20 TeV", nbin=10)
+energy_axis_true = MapAxis.from_energy_bounds(
+    "0.1 TeV", "40 TeV", nbin=20, name="energy_true"
+)
+
+on_region_radius = Angle("0.11 deg")
+on_region = CircleSkyRegion(center=target_position, radius=on_region_radius)
+
+geom = RegionGeom.create(region=on_region, axes=[energy_axis])
+
+dataset_maker = SpectrumDatasetMaker(
+    containment_correction=True, selection=["counts", "exposure", "edisp"]
+)
+bkg_maker = ReflectedRegionsBackgroundMaker()
+safe_mask_masker = SafeMaskMaker(methods=["aeff-max"], aeff_percent=10)
+
+datasets = Datasets()
+
+dataset_empty = SpectrumDataset.create(geom=geom, energy_axis_true=energy_axis_true)
+
+for obs in short_observations:
+    dataset = dataset_maker.run(dataset_empty.copy(), obs)
+
+    dataset_on_off = bkg_maker.run(dataset, obs)
+    dataset_on_off = safe_mask_masker.run(dataset_on_off, obs)
+    datasets.append(dataset_on_off)
+
+spectral_model = PowerLawSpectralModel(
+    index=3.4, amplitude=2e-11 * u.Unit("1 / (cm2 s TeV)"), reference=1 * u.TeV
+)
+spectral_model.parameters["index"].frozen = False
+
+sky_model = SkyModel(spatial_model=None, spectral_model=spectral_model, name="pks2155")
+
+datasets.models = sky_model
+
+lc_maker_1d = LightCurveEstimator(
+    energy_edges=[0.7, 20] * u.TeV,
+    source="pks2155",
+    time_intervals=time_intervals,
+    selection_optional=None,
+)
+
+lc_1d = lc_maker_1d.run(datasets)
+
+plt.figure(figsize=(8, 6))
+lc_1d.plot(marker="o")
+plt.show()
+
+######################################################################
+# # Methods to characterize variability
+# ---------------
+#  Amplitude maximum variation, relative variability amplitude and variability amplitude
+#
+# The amplitude maximum variation is the simplest method to define variability ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2016A&A...588A.103B/abstract)) as it just computes the level of tension between the lowest and highest measured fluxes in the lighcurve
+
+flux = lc_1d.flux.data
+flux_err = lc_1d.flux_err.data
+
+f_mean = np.mean(flux)
+f_mean_err = np.mean(flux_err)
+
+f_max = flux.max()
+f_max_err = flux_err[np.argmax(flux)]
+f_min = flux.min()
+f_min_err = flux_err[np.argmin(flux)]
+
+ampl_max = (f_max - f_max_err) - (f_min - f_min_err)
+
+ampl_sig = ampl_max / np.sqrt(f_max_err**2 + f_min_err**2)
+
+print(ampl_sig)
```

Comment:
Maybe define functions `maximal_amplitude_variation`, `relative_amplitude_variation`, etc

Reply:
Agree, they are not very clear like this. Fixing it.